app/layouts/root_layout/root_layout.js

```diff
@@ -9,9 +9,33 @@ import {IntlProvider} from 'react-intl';
 export default class RootLayout extends React.Component {
     static propTypes = {
         children: React.PropTypes.node,
-        locale: React.PropTypes.string.isRequired
+        locale: React.PropTypes.string.isRequired,
+        storage: React.PropTypes.object,
+        users: React.PropTypes.object,
+        actions: React.PropTypes.shape({
+            goToSelectServer: React.PropTypes.func,
+            goToSelectTeam: React.PropTypes.func,
+            setStoreFromLocalData: React.PropTypes.func
+        }).isRequired
     };
 
+    constructor(props) {
+        super(props);
+        this.alreadyLogged = false;
+    }
+
+    componentWillReceiveProps(nextProps) {
```

Comment:
And how should we avoid showing the first scene if we want to move to another one? I couldn't find a better solution for that, maybe we can talk about this